queries.sql

```diff
@@ -21,3 +21,71 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equal precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- ---------------- --
+-- SECOND MILESTONE --
+-- ---------------- --
+
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+-- Verify that changes were made.
+-- Commit the transaction.
+-- Verify that changes persist after commit.
+
+BEGIN;
+UPDATE animals SET species = 'digimon ' WHERE NAME LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+SELECT * FROM animals ORDER BY id;
```

Comment:
- [ ] Similarly, let's verify that changes are verified before and after commit.